Kitodo/src/main/java/de/sub/goobi/forms/SpracheForm.java

```diff
@@ -135,6 +161,15 @@ public void switchLanguage(String langCodeCombined) throws IOException {
         } else {
             locale = new Locale(languageCode[0]);
         }
+        try {
+            User user = serviceManager.getUserService().getById(serviceManager.getUserService().getAuthenticatedUser().getId());
```

Comment:
I think this line is too long. Did you apply the formatting rules?

Reply:
Yes, I did.
It 's not too long according to the CheckStyle.

Reply:
Checkstyle allow lines with 140 characters and this line is below. So it should be ok for now.